packages/now-next/src/index.ts

```diff
@@ -223,9 +224,17 @@ export const build = async ({
     }
   }
 
+  if (hasFunctionsConfig) {
+    console.warn(
+      `WARNING: your application is being opted out of @vercel/next's optimized lambdas mode due to \`functions\` config. http://err.sh/vercel/vercel/next-functions-config-optimized-lambdas`
```

Comment:
What do you think about putting the link on a new line?

```suggestion
      `WARNING: Your application is being opted out of "@vercel/next" optimized lambdas mode due to \`functions\` config.\nMore info: http://err.sh/vercel/vercel/next-functions-config-optimized-lambdas`
```

Reply:
Sounds good to me